src/org/usfirst/frc4904/robot/commands/AutoShifter.java

```diff
@@ -0,0 +1,68 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import java.util.LinkedList;
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.custom.sensors.CustomEncoder;
+import org.usfirst.frc4904.standard.subsystems.chassis.SolenoidShifters;
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class AutoShifter extends Command {
+	public CustomEncoder leftEncoder;
+	public CustomEncoder rightEncoder;
+	public SolenoidShifters shifter = RobotMap.Component.shifter;
```

Comment:
Please be consistent in how this command takes in components - in this case, use them all in the constructor. In fact, why not take in a whole `ShiftingChassis`?